README.md

```diff
@@ -104,35 +98,20 @@ class ApiHealthCheck implements HealthCheckInterface
 registered health checkers. If everything was OK, just return the report that was passed to the method. 
 
 ### Register the checker
-To actually include the home made checker simply tag it with 'surfnet.monitor.health_check'
-
-Example service definition in `services.yml`
-
-```yaml
-services:
-    acme.monitor.my_custom_health_check:
-        class: Acme\AppBundle\HealthCheck\MyCustomHealthCheck
-        arguments:
-            - @test_service
-        tags:
-            - { name: surfnet.monitor.health_check }
-```
+By implementing the `HealthCheckInterface` you can register your own health check. 
+This interface is tagged with `openconext.monitor.health_check` so you don't have to do it yourself.
```

Comment:
Now this is what I call progress!

Reply:
One thought tho. Quite some people now use this old construct. Have you verified that stuff keeps working without them changing any code? If a change is needed, this PR should trigger a new major release as we are breaking BC. Also in that case. A small readme on how to migrate to the new version is a must.

But as I see it from my code review chair, I think your change does not cause a BC break. But please check this!

Reply:
The only thing is the path to the routes. They are changed from Resources to /config

We can mention that in the release notes?

Reply:
Rule of thumb; if any changes need to be done by the consumer due to a change made in a new release. We should provide an `upgrading.md` entry. Describing the way you can port from the previous version to the new way of doing things.

If there is no BC breakage, there is no need to document it.